### ISSUE TITLE
Enhancement of the landing page

### ISSUE THREAD

**Issue by Moses-main** [Moses-main/Brain-Gain#3]

**Comment by Khushi-Gupta13** [comment]
I would like to add modifications to the home page of the website as my contribution to hackotber fest. Kindly assign me with the task.

**Comment by sakshi292** [comment]
I would like to do the changes as I am contributing in hacktoberfest please assign me this issue.

**Comment by Moses-main** [comment]
@sakshi292 could you create another issue use a title of what you'd want to do so that i can assign the task to you.

**Comment by sakshi292** [comment]
I can do the modifications in text interfaces and page texture and layouts of the site as  I am doing Html and CSS

**Comment by Moses-main** [comment]
@sakshi292 wouyld you be doing it for all the site ? Hope you've gone through the platform and seen how it looks

This is the link to the demo of the site https://moses-main.github.io/Brain-Gain/

**Comment by sakshi292** [comment]
No, I am not doing the same things for all the sites.Just exploring my html as your site is based on html or css

**Comment by Moses-main** [comment]
I have created another issue with you on that. So get ready to contribute to the one i'd assign to you. You have to leave this one for [Khushi-Gupta13].

**Comment by Khushi-Gupta13** [comment]
Thank you for assigning me the task @ *Moses-main
<https://github.com/Moses-main>*

**Comment by Moses-main** [comment]
You're welcome @Khushi-Gupta13

**Comment by Khushi-Gupta13** [comment]
Hi @Moses-main <https://github.com/Moses-main>  I'm not able to edit the
files present in your git hub. Kindly provide me the access.
